serverless-http.js

```diff
@@ -16,6 +16,10 @@ module.exports = function(app, opts) {
   const options = Object.assign({}, defaultOptions, opts);
 
   return (evt, ctx, callback) => {
+    if (options.callbackWaitsForEmptyEventLoop != null) {
```

Comment:
I would be interested to understand why (in the final code) callbackWaitsForEmptyEventLoop is always set to false?
Having it as an option (as was proposed) makes total sense, but I don't understand why you would always set it to false? Is there something in express/koa/... that adds events to the event loop even though the request is fully processed and the response can go to the client?
Thanks!

(Please note: I have very limited experience with express/koa - but I have quite a bit with lambda, and so far, found very few reasons to allow the instance to freeze with a non-empty event loop...?)